CHANGELOG.md

```diff
@@ -19,6 +19,7 @@ All notable changes to this project will be documented in this file. The format
   - Detroit AMI data and import script
     ([#443](https://github.com/CityOfDetroit/bloom/pull/443))
   - Fetch UnitsSummary data in listing query ([#441](https://github.com/CityOfDetroit/bloom/pull/441))
+  - UI for adding / editing units summaries ([#475](https://github.com/CityOfDetroit/bloom/pull/475))
```

Comment:
I think you should put this under the M11 milestone, since that's when it was merged (instead of when it was scheduled).